scripts/ldev2pcs

```diff
@@ -102,10 +125,29 @@ def configure_location(resource, nodes, fixed_score_string=None):
             score -= 10
 
 def configure():
-    cmd = 'pcs cluster setup --force --local'
-    cmd += ' --name '+cmdline_args.cluster_name
-    cmd += ' '+cmdline_args.mgmt_node
-    run(cmd)
+    # the syntax for making the corosync.conf
+    # file depends on the pacemaker version
+    pacemaker_major_number = get_pacemaker_major_number()
+    if pacemaker_major_number == 1:
+        cmd = 'pcs cluster setup --force --local'
+        cmd += ' --name '+cmdline_args.cluster_name
+        cmd += ' '+cmdline_args.mgmt_node
+    if pacemaker_major_number == 2:
+        cmd = 'pcs cluster setup'
+        cmd += ' '+cmdline_args.cluster_name
+        cmd += ' '+cmdline_args.mgmt_node
+        cmd += ' --corosync_conf /etc/corosync/corosync.conf --overwrite'
+    if not cmdline_args.no_corosync:
```

Comment:
Should some other major number generate an exception?

Reply:
Good point, I'll change it to throw an exception in the case that the major number is not 1 or 2, since I don't now how to deal with other cases.